SU-SWS/linky_clicky
# SITES-343:Ensure that User One is blocked from entering PHP

# READY FOR REVIEW

# Summary
- Ensure that User 1 cannot enter PHP code

# Needed By (Date)
- February 15th (end of sprint)

# Criticality
- How critical is this PR on a 1-10 scale? 8ish

# Steps to Test

1. Check out this branch
2. Create a new site on ACSF `dev` or `test` environment
3. Enable the `paranoia` module
4. Run Behat with `php-userone.feature`
5. Run Behat with `php-administrator.feature`
6. Ensure that all Scenarios pass

Note: I am intentionally *not* enabling `paranoia` in these Features because of https://stanfordits.atlassian.net/browse/SITES-330 (we want to enforce Paranoia elsewhere)

# Affected Projects or Products
- ACSF

# Associated Issues and/or People
- JIRA ticket: https://stanfordits.atlassian.net/browse/SITES-343
